src/app/shared/admins/services/admins.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from "@angular/core";
+import {
+  MatSnackBar,
+  MatSnackBarRef,
+  SimpleSnackBar
+} from "@angular/material/snack-bar";
+import { environment } from "@environment";
+import { Store } from "@ngxs/store";
+import { AWSError, CognitoIdentityServiceProvider } from "aws-sdk";
+import {
+  ListUsersInGroupRequest,
+  ListUsersResponse
+} from "aws-sdk/clients/cognitoidentityserviceprovider";
+import { GetError, GetSuccess } from "../state/admins.actions";
+
+@Injectable({
+  providedIn: "root"
+})
+export class AdminsService {
+  constructor(private store: Store, private snackBar: MatSnackBar) {}
+
+  public get listUsersInGroupRequest(): ListUsersInGroupRequest {
+    return {
+      UserPoolId: environment.awsConfig.userPoolId,
+      GroupName: "site-admin-group"
```

Comment:
this looks like it could be re-usable function which accepts `groupName` as parameter. Or could be hard-coded in `getAdminUsers` as the function name implies

Reply:
*Note group names have been slightly altered and may incorporate multiple rules `https://hee-tis.atlassian.net/wiki/spaces/NTCS/pages/1671495738/Revalidation+-+Access+Permissions`